source/options.css

```
@@ -130,6 +131,17 @@ h2 ~ h2 {
 	margin-left: 0.6em;
 }
 
+.feature-new .feature-name::after {
+	content: 'New';
+	color: var(--feature-new);
+	font-size: 14px;
+	border: 1px solid var(--feature-new);
```

`border` inherits from `color`

```suggestion
	border: 1px solid;
```